### DISCUSSION TITLE
How to implement expanding collectionview in MAUI?

### DISCUSSION THREAD

**Discussion by devonuto** [dotnet/maui#12234] [Q&A]
Looking for a way to implement and expanding list view in my app.  I found the CommunityToolkit expander, but that says not to use it in CollectionViews.  

With the current state of MAUI, is this possible? If so, how are people doing this?

An example of this being done in C# would be excellent if someone has one.  TIA.

Cheers.

**Comment by VladislavAntonyuk** [comment]
Hello @devonuto. We have a PR to add support for CollectionView.
For now you can copy the code to your project: https://github.com/CommunityToolkit/Maui/pull/751
